lib/index.js

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const brLedger = require('bedrock-ledger');
+const equihashSigs = require('equihash-signature');
+const validate = require('bedrock-validation').validate;
+const BedrockError = bedrock.util.BedrockError;
+
+require('./config');
+
+bedrock.events.on('bedrock.start', () => brLedger.use('EquihashValidator2017', {
+  type: 'validator',
+  api: api
+}));
+
+const api = {};
+// NOTE: only exported for tests
```

Comment:
Ok, I guess so. It seemed like from the README that you could just require it (see Usage).